packages/tags/src/tag.ts

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import {
+    html,
+    LitElement,
+    CSSResultArray,
+    TemplateResult,
+    property,
+    PropertyValues,
+} from 'lit-element';
+
+import '@spectrum-web-components/button';
+
+import styles from './tag.css.js';
+
+/**
+ * @element sp-tags
+ */
+export class Tag extends LitElement {
+    public static get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    @property({ type: Boolean, reflect: true })
+    public deletable = false;
+
+    @property({ type: Boolean, reflect: true })
+    public disabled = false;
+
+    constructor() {
+        super();
+        this.addEventListener('focusin', this.handleFocusin);
+    }
+
+    private handleFocusin = (): void => {
+        this.addEventListener('focusout', this.handleFocusout);
+        this.addEventListener('keydown', this.handleKeydown);
+    };
+
+    private handleFocusout = (): void => {
+        this.removeEventListener('keydown', this.handleKeydown);
+        this.removeEventListener('focusout', this.handleFocusout);
+    };
+
+    private handleKeydown = (event: KeyboardEvent): void => {
+        if (!this.deletable) {
+            return;
+        }
+        const { code } = event;
+        switch (code) {
+            case 'Backspace':
+            case 'Space':
+            case 'Delete':
+                this.delete();
+                return;
+            default:
+                return;
+        }
+    };
+
+    private delete(): void {
+        this.dispatchEvent(
+            new Event('delete', {
+                bubbles: true,
+            })
+        );
+    }
+
+    protected render(): TemplateResult {
+        return html`
+            <slot name="avatar"></slot>
+            <slot name="icon"></slot>
+            <span class="label"><slot></slot></span>
+            ${this.deletable
+                ? html`
```

Comment:
I'm excited that we're getting the `sp-clear-button` pattern in! 

I've been using `sp-action-button quiet` for all my clear button needs